tap_mysql/__init__.py

```diff
@@ -484,10 +484,10 @@ def sync_table(connection, catalog_entry, state):
         if replication_key_value is not None:
             if catalog_entry.schema.properties[replication_key].format == 'date-time':
                 replication_key_value = pendulum.parse(replication_key_value)
-                select += ' WHERE `{}` >= %(replication_key_value)s ORDER BY `{}` ASC'.format(
-                    replication_key,
-                    replication_key)
-                params['replication_key_value'] = replication_key_value
+            select += ' WHERE `{}` >= %(replication_key_value)s ORDER BY `{}` ASC'.format(
```

Comment:
That's fantastic. The indentation being off caused this? Don't tell @psantacl ...

Reply:
@mdelaurentis Yeah...emacs's `indent-region` in Python mode actually seems to move lines following an `if` to be indented into that block...

![indent region](https://user-images.githubusercontent.com/1185129/30757179-b015818a-9f9b-11e7-90f6-680087f43b43.gif)

Reply:
unless there's a blank line before the part that shouldn't be indented, apparently